docs/schemaValidationMiddleware.md

```diff
@@ -213,7 +213,7 @@ my.kettle.handler.reportSuccess = function (request) {
 
 fluid.defaults("my.kettle.validator", {
     gradeNames: ["fluid.schema.kettle.validator.body"],
-    inputSchema: {
+    requestSchema: {
```

Comment:
From my experience you also needed to specify the `$schema` property. e.g. `"$schema": "fss-v7-full#",` as this doesn't merge.